spec/intro_to_ruby_hashes_lab_spec.rb

```diff
@@ -1,134 +1,110 @@
+        end
 
-        expect(monopoly_with_third_tier.keys.count).to eq(1)
-        expect(monopoly_with_third_tier.values[0].values.count).to eq(3)
-        expect(monopoly_with_third_tier.values[0][:names].keys[2]).to eq(:b_and_o_railroad)
-        expect(monopoly_with_third_tier.values[0][:names].values[2]).to eq({})
-      end
+        it "has a key :pennsylvania_railroad that points to an empty Hash" do
+          expect(monopoly_with_third_tier[:railroads][:names].has_key?(:pennsylvania_railroad)).to(be_truthy, "key :pennsylvania_railroad should be present in this Hash")
+          expect(monopoly_with_third_tier[:railroads][:names][:pennsylvania_railroad]).to(eq({}), ":pennsylvania_railroad should point to an empty Hash")
+        end
 
-      it "sets the 4th key of the :names hash to a symbol, :shortline, whose value is an empty hash" do
+        it "has a key :b_and_o_railroad that points to an empty Hash" do
+          expect(monopoly_with_third_tier[:railroads][:names].has_key?(:b_and_o_railroad)).to(be_truthy, "key :b_and_o_railroad should be present in this Hash")
+          expect(monopoly_with_third_tier[:railroads][:names][:b_and_o_railroad]).to(eq({}), ":b_and_o_railroad should point to an empty Hash")
+        end
 
-        expect(monopoly_with_third_tier.keys.count).to eq(1)
-        expect(monopoly_with_third_tier.values[0].values.count).to eq(3)
-        expect(monopoly_with_third_tier.values[0][:names].keys[3]).to eq(:shortline)
-        expect(monopoly_with_third_tier.values[0][:names].values[3]).to eq({})
+        it "has a key :shortline_railroad that points to an empty Hash" do
+          expect(monopoly_with_third_tier[:railroads][:names].has_key?(:shortline_railroad)).to(be_truthy, "key :shortline_railroad should be present in this Hash")
+          expect(monopoly_with_third_tier[:railroads][:names][:shortline_railroad]).to(eq({}), ":shortline_railroad should point to an empty Hash")
+        end
       end
-    end
   end
 
-  describe "Adding Key/Value Pairs in the Fourth Level" do
-    describe "monopoly_with_fourth_tier" do
-      it "sets the 1st key of the :reading_railroad hash to a string, 'mortgage_value', whose value is also a string, '$100'" do
-
-        expect(monopoly_with_fourth_tier.values[0][:names].has_key?(:reading_railroad)).to be true
-        expect(monopoly_with_fourth_tier.values[0][:names].values[0].has_key?("mortgage_value")).to be true
-        expect(monopoly_with_fourth_tier.values[0][:names].values[0].values[0]).to eq("$100")
-      end
-
-      it "sets the 1st key of the :pennsylvania_railroad hash to a string, 'mortgage_value', whose value is also a string, '$200'" do
+  describe "monopoly_with_fourth_tier method builds on the hash returned by monopoly_with_third_tier, but updates the Hash pointed at by the :railroads :names pairs such that the railroad name key points to a Hash with a key \"mortgage_value\" that is a String with a $-sign and an Integer after it" do
```

Comment:
'updates the Hash pointed at by the :railroads :names pairs' sounds a little confusing. 

updates the Hash so that each of the :railroads :names keys points to a Hash with a key...